src/lib/symmetric.c

```diff
@@ -124,8 +124,10 @@ pgp_cipher_start(pgp_crypt_t *crypt, pgp_symm_alg_t alg, const uint8_t *key, con
     memset(crypt, 0x0, sizeof(*crypt));
 
     const char *cipher_name = pgp_sa_to_botan_string(alg);
-    if (cipher_name == NULL)
+    if (cipher_name == NULL) {
+        fprintf(stderr, "Unsupported algorithm: %d\n", alg);
```

Comment:
Can you use RNP_LOG() instead?

Reply:
`symetric.c` hasn't got any of using `RNP_LOG`, so, I think the right way is rewrite all `fprintf` to `RNP_LOG`